packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -449,7 +449,11 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
           objectMetadataForUpdate,
         );
 
-      // TODO: recompute foreign keys indexes as well (in the related object and not objectMetadataForUpdate)
+      await this.objectMetadataMigrationService.recomputeEnumNames(
```

Comment:
very nit - naming: "recompute" does not necessarily entice "migrate" (in my mind)

Reply:
I think the `objectMetadataMigrationService` service name should be explicit enough otherwise would make it redundant (if we say we only do migrations in that service but I believe that's the case. 
Wdyt @ijreilly?